coursedata/level-defaults/de.yaml

```diff
@@ -1,16 +1,11 @@
 1:
     intro_text: |
-        Willkommen bei Hedy! In Level 1 kannst du die Befehle `print`, `ask` und `echo` verwenden. 
-
-        Hedy benutzt englische Wörter für Befehle wie die meisten Programmiersprachen. `print` bedeutet drucken oder auf den Bildschirm ausgeben, `ask` bedeutet fragen und `echo` bedeutet wiederholen.
-
-        Auf der linken Seite deines Bildschirms befinden sich alle Befehle des Level 1.
-
-        Drücke den blauen Knopf und der Befehl wird in dein Programm übernommen!
-
+        ## Willkommen bei Hedy!
+        In Level 1 kannst du die Befehle `print`, `ask` und `echo` verwenden.
+        Tippe den Code in das Programmierfeld. Oder drücke den grünen Knopf beim Beispielcode um ihn in dein Programm zu übernehmen!
```

Comment:
As we translate from the old `de` version to another version we are missing the keyword placeholders? I'm not sure how this is possible as I manually fixed all these placeholders when implementing the keyword languages. We should not merge this before this is fixed as otherwise our keyword language is not supported.

Do you have an idea how this is possible? @Mark-Giesen @Felienne